client/components/jetpack-header/index.jsx

```diff
@@ -22,7 +22,7 @@ export class JetpackHeader extends PureComponent {
 
 	static propTypes = {
 		partnerSlug: PropTypes.string,
-		width: PropTypes.number,
+		width: PropTypes.oneOfType( [ PropTypes.number, PropTypes.string ] ),
```

Comment:
We won't need to support a string `width` prop if we apply the width/max-width or apply some padding/margin by using CSS.